app/Actions/Diagnostics/Checks/IniSettingsCheck.php

```diff
@@ -51,9 +48,8 @@ public function check(array &$errors): void
 			$errors[]
 				= 'Warning: You may experience problems when uploading a photos of large size. Take a look in the FAQ for details.';
 		}
-		if (
-			intval(ini_get('max_execution_time')) < 200
-		) {
+		$max_execution_time = intval(ini_get('max_execution_time'));
+		if (0 < $max_execution_time && $max_execution_time < 200) {
 			$errors[]
 				= 'Warning: You may experience problems when uploading a large amount of photos. Take a look in the FAQ for details.';
```

Comment:
I know this isn't one of the lines you changed, but I have a problem with it. Basically, the diagnostics is bogus. The value of `max_execution_time` is irrelevant to the success of uploading multiple photos. Photos are uploaded one-per-API-call. It used to matter for _import_, but these days we manually reset it for every imported photo so it's irrelevant there as well.

Basically, as far as photos are concerned, the value needs to be set to something long enough to process a _single_ photo. I would imagine that for any remotely sensible server, `30` would be enough, though I guess for some low-end RPi more may be needed?

The limit is more likely to be hit if somebody has thousands of albums than during photo upload/import...

Reply:
> 30 would be enough, though I guess for some low-end RPi more may be needed?

Or... very very big pictures... :see_no_evil:

Reply:
Is this conversation resolved? Can we merge?